### DISCUSSION TITLE
Teamspeak 3 Gateway

### DISCUSSION THREAD

**Discussion by mightyBroccoli** [SpectrumIM/spectrum2#411] [Ideas]
I am interested in a gateway from Teamspeak3 to XMPP.
There is a framework that is highly functional and is 100% supported and updated in short timeframes. 
We are using a big teamspeak server and we would love a gateway to chat with ppl/ channels of ppl and the framework supports that completly there is even the php code out but I am not capable of coding stuff like that. 
Is there hope that you guys do that? There are so many forums asking for this stuff but noone coded anything to this date..

**Comment by vitalyster** [comment]
which framework you are mentioned?

**Comment by mightyBroccoli** [comment]
Oh I totally forgot to link the pages. I am sorry.
First the framework [TS3 PHP Framework](https://docs.planetteamspeak.com/ts3/php/framework/) and secondly the codes from users from the official teamspeak forums [Teamspeak Forum](http://forum.teamspeak.com/threads/54132-API-TS3-PHP-Framework/page30)

**Comment by mightyBroccoli** [comment]
Do I guess right that this means this will be developed?

**Comment by fank** [comment]
@mightyBroccoli this means only that this issue is a feature and not a bug or something else.
